### DISCUSSION TITLE
CMIChatColor support.

### DISCUSSION THREAD

**Discussion by Merunko** [BG-Software-LLC/SuperiorSkyblock2#1828] [Pending Feature Requests]
I managed to modify the mission handler jar to support CMIChatColor.translate() method from CMILib for hex color support in the missions configuration file.
![image](https://github.com/BG-Software-LLC/SuperiorSkyblock2/assets/141312536/31fc41b7-f6ef-4adc-9b98-b8c73c4fa989)

Now who have any idea how to have this feature for other configuration file like missions-category,yml, missions.yml, Any configuration files that require "name", "title", "lore". This should be a new feature for better interface. Or anyone here who's kindly enough that can make a custom jar to give this feature.

**Comment by Merunko** [comment]
Or maybe a new placeholder?

**Comment by Ceduz19** [comment]
Hex color support is already implemented, you just need to use the format _&{HEX:ffffff}_

**Reply by Merunko** [reply]
Omg thank you so much. How about gradient color? CMIChatColor can translate a gradient color format {#FFFFFF>}text{#101010<}
